glommio/src/executor/placement/mod.rs

```diff
@@ -214,23 +248,76 @@ impl CpuSet {
 
     /// Returns a reference to the [`CpuLocation`]s currently included in the
     /// `CpuSet`.
-    pub fn as_vec(&self) -> &Vec<CpuLocation> {
-        &self.0
+    pub fn as_vec(&self) -> Vec<&CpuLocation> {
+        self.0.iter().collect()
```

Comment:
I don't think returning a vector of references is the right thing to do here. Since we changed the internal representation from a Vec to something else I'm not even sure this function makes sense anymore + you added an iterator interface that achieves the same thing.

Reply:
I removed the caller; should I remove this one too?

Reply:
Please do, `.iter()` can be used to achieve the same result.